src/commands/import.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright 2014-2017 The Rooster Developers
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use getopts;
+use serde_json;
+use password::v2::{Password, PasswordStore};
+use std::fs::File;
+use std::io::Write;
+
+pub fn callback_help() {
+    println!("Usage:");
+    println!("    rooster import -h");
+    println!("    rooster import <file_path>");
+    println!();
+    println!("Example:");
+    println!("    rooster import dump.json");
+}
+
+pub fn callback_exec(_matches: &getopts::Matches, store: &mut PasswordStore) -> Result<(), i32> {
```

Comment:
Replace `_matches` with `matches`. I used `_matches` in some places to suppress warning when this variable was unused. Here, the variable is used so the `_` is not needed.